src/cpp/core/r_util/RSourceIndex.cpp

```diff
@@ -559,6 +559,9 @@ RSourceIndex::RSourceIndex(const std::string& context, const std::string& code)
    inferredPkgNames_.clear();
 
    // tokenize and create token cursor
+   if (boost::algorithm::contains(code, "do not edit by hand"))
+      return;
```

Comment:
This only deals with R files, e.g. `cpp11.R` but the generated C++ files usually contain either decalarations, or `extern "C"` prefixed definitions, and both are already ignored, so it does not feel necessary to prevent their indexing.